<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigObjectTableView</class>
 <widget class="QWidget" name="ConfigObjectTableView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableView" name="tableView">
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="moveWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="itemTop">
          <property name="toolTip">
           <string>Move selected item(s) to the top.</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="edit-move-top"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="itemTenUp">
          <property name="toolTip">
           <string>Move selected item(s) ten positions up.</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="edit-move-10-up"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="itemUp">
          <property name="toolTip">
           <string>Move selected item(s) one position up.</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="edit-move-up"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="itemDown">
          <property name="toolTip">
           <string>Move selected item(s) one position down.</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="edit-move-down"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="itemTenDown">
          <property name="toolTip">
           <string>Move selected item(s) ten positions down.</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="edit-move-10-down"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="itemBottom">
          <property name="toolTip">
           <string>Move selected item(s) to the bottom.</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="edit-move-bottom"/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="filterWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="filterToggleButton">
        <property name="text">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="filterEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="clearButton">
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::EditClear"/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionToggleFilterSort">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="item-filter"/>
   </property>
   <property name="text">
    <string>toggle Filter and Sorting</string>
   </property>
   <property name="toolTip">
    <string>Toggle Filter and Sorting</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+L</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionCloseSortFilter">
   <property name="text">
    <string>Close Sort and Filter</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
